src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is starting its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator has stopped working.");
```

Comment:
Similarly, the stop message should be more informative. Make sure it clearly indicates that it is an `Excavator` that has stopped working.